lib/generators/atomic_cms/assets/assets_generator.rb

```diff
@@ -1,6 +1,23 @@
 module AtomicCms
   module Generators
     class AssetsGenerator < Rails::Generators::Base
+      def install_bourbon
+        gem 'bourbon', '~> 4.2.6'
+      end
+
+      def install_neat
+        gem 'neat', '~> 1.7.2'
+      end
+
+      def install_bitters
+        gem 'bitters', '~> 1.1.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.